add_balancer.yml

```diff
@@ -157,9 +157,7 @@
       when: with_haproxy_load_balancing|bool
 
     - role: confd
-      when: dcs_type == "etcd" and
-        with_haproxy_load_balancing|bool
+      when: dcs_type == "etcd" and with_haproxy_load_balancing|bool
 
     - role: keepalived
-      when: cluster_vip is defined and cluster_vip | length > 0 and
-        with_haproxy_load_balancing|bool
+      when: cluster_vip is defined and cluster_vip | length > 0 and with_haproxy_load_balancing|bool
```

Comment:
perhaps the best practice would be to switch to lists so as not to make the string too long for large conditions

when:
  - condition1
  - condition2